tests/test_on_data/ecephys/test_spikeglx_converter.py

```diff
@@ -251,8 +283,10 @@ class TestSortedSpikeGLXConverter:
     def test_multi_probe_multi_stream_example(self, tmp_path):
         """dataset with two probes and both ap and lf streams"""
         # Initialize base SpikeGLX converter (notebook example 1)
+        # Disable sync channels since this test focuses on sorted units
         spikeglx_converter = SpikeGLXConverterPipe(
-            folder_path=SPIKEGLX_PATH / "multi_trigger_multi_gate" / "SpikeGLX" / "5-19-2022-CI0"
+            folder_path=SPIKEGLX_PATH / "multi_trigger_multi_gate" / "SpikeGLX" / "5-19-2022-CI0",
+            # include_sync_channels removed - use streams parameter to exclude
```

Comment:
I don't see any tests that actually use the streams parameter. Maybe I'm missing something, but that seems like a pretty critical parameter that should be tested.

Reply:
I agree, there should be one - and preferably only one - obvious way to do things. 

Even if we remove the streams parameter later, though, I think it should be tested in the interim. Since its behavior was changed by this PR, I think it's within the scope.

Reply:
I am fine with adding test but where is that the behavior was changed? So I can focus the test on that.

Reply:
Ah, I see. Yes, before we did not have the feature and they were not included `SpikeGLXSyncChannelInterface`. Then, following the principle that the converter `SpikeGLXConverterPipe` should include EVERYTHING I aded it there.

Now, within the development of this branch, [at the start](https://github.com/catalystneuro/neuroconv/pull/1600/commits/913325744a371557e2d3d2a06c8d9c98cf471259), I did not add the sync channel to the convert by default. But then I thought that the converter should add everything available and that people could exclude that by using either of the two ways mentioned above so I eliminated that aforementioned bool.